Lib/test/test_launcher.py

```diff
@@ -717,3 +717,36 @@ def test_literal_shebang_invalid_template(self):
             f"{expect} arg1 {script}",
             data["stdout"].strip(),
         )
+
+    def test_shebang_command_in_venv(self):
```

Comment:
This test was backported from 3.13 but _might_ not work here. If we didn't make the relevant fix, probably best to just not backport the test.